.env-dist

```diff
@@ -20,13 +31,13 @@ APP_ENV=local
 ## Use 'php artisan key:generate' to generate a new key. Key size must be 16, 24 or 32.
 APP_KEY=
 #APP_LOCALE=en
-## LOG setting. Where and/or how the log file is setup. Options are single (default), daily, syslog, errorlog
-#APP_LOG=single
+## Here you may configure the log channels for your application. Options are stack (default), slack, single, daily, syslog, errorlog
+#LOG_CHANNEL=stack
 ## LOG Level. This is hierarchical and goes in the following order.
 ## DEBUG -> INFO -> NOTICE -> WARNING -> ERROR -> CRITICAL -> ALERT -> EMERGENCY
 ## If you set log level to WARNING then all WARNING, ERROR, CRITICAL, ALERT, and EMERGENCY
 ## will be logged. Setting log level to DEBUG will log everything.
-#APP_LOG_LEVEL=warning
+APP_LOG_LEVEL=warning
```

Comment:
Shouldn't we leave it commented? so then in the config, it takes default value.

Reply:
@yaroslavmo, yes. you are right. I fixed this.